Makefile

```diff
@@ -69,23 +70,27 @@ code/fix:
 
 .PHONY: image/build
 image/build: code/compile
-	@operator-sdk build $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	@operator-sdk build $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: image/push
 image/push:
-	docker push $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	docker push $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: image/build/push
 image/build/push: image/build image/push
 
 .PHONY: image/build/test
 image/build/test:
-	operator-sdk build --enable-tests $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	operator-sdk build --enable-tests $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: test/unit
 test/unit:
 	TEMPLATE_PATH=$(TEMPLATE_PATH) ./scripts/ci/unit_test.sh
 
+.PHONY: test/e2e/prow
+test/e2e/prow: INTEGREATLY_OPERATOR_IMAGE := "registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:integreatly-operator"
```

Comment:
Does this need export as well?

Reply:
I wouldn't consider that one a blocker for now, since it's not being used, and there are no downstream targets of this. Since we know we'll need another change after the openshift/release one, we can export this in that also.